pyodide-build/pyodide_build/io.py

```diff
@@ -103,20 +80,123 @@ def check_package_config(
                 # check types
                 continue
             if not isinstance(value, expected_type):
-                errors_msg.append(
+                yield (
                     f"Wrong type for '{section_key}/{subsection_key}': "
                     f"expected {expected_type.__name__}, got {type(value).__name__}."
                 )
 
-    if raise_errors and errors_msg:
+
+def _cpc_source(config: dict[str, Any]) -> Iterator[str]:
```

Comment:
What's "cpc"? maybve "check" or "validate"?

**Edit:** ah, I see check_package_config. Maybe `_check_config_source` to be more explicit